services/api/mw2pdf/utils/validation.js

```diff
@@ -0,0 +1,31 @@
+import Joi from 'joi'
+
+export function assertValidPassthroughParameters(passthroughParameters) {
+  Joi.assert(
+    passthroughParameters,
+    Joi.string()
+      .custom(
+        (value, helpers) => {
+          JSON.parse(value)
```

Comment:
So, if `JSON.parse` throws an error, then the this `custom` validator will throw an error, too, which will _bubble up_ and cause the assert to fail? I just want to make sure that I am understanding this correctly?

Reply:
That irony when the validation logic is not actually explicit 😂

Yes exactly -- I may add a comment explaining that's what's going on here

Reply:
I made the error catching flow more explicit (and took the opportunity to make the resulting message a bit more clear too)